src/main/resources/schema.sql

```diff
@@ -6,3 +6,16 @@ CREATE TABLE IF NOT EXISTS SAMPLE(
   deleted      BIT         DEFAULT 0,
   PRIMARY KEY (id)
 );
+
+CREATE TABLE IF NOT EXISTS USERS(
+  id           BIGINT(50)  NOT NULL AUTO_INCREMENT,
+  firstName  VARCHAR(50) DEFAULT NULL,
```

Comment:
firstName, lastName y password deberían ser NOT NULL según lo que esta definido en la entity.

Reply:
actualizado